contrib/inventory/azure_rm.py

```diff
@@ -187,14 +187,18 @@
 '''
 
 import argparse
-import ConfigParser
 import json
 import os
 import re
 import sys
 import inspect
-import traceback
 
+try:
+    # python2
+    import ConfigParser as cp
+except ModuleNotFoundError:
```

Comment:
I htink you need ImportError here.  ModuleNotFoundError isn't a standard exception on Python-3.5

Reply:
Great point, done!